tests/test-recipes/metadata/noarch_python/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: noarch_python_test_package
+  version: "1.0"
+
+source:
+  path: ./noarch_python_test_package
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: python
+  entry_points:
+    - noarch_python_test_package_script = noarch_python_test_package:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+# Tests are commented out because they should fail since conda does not yet know how to install
```

Comment:
I think it's better to keep these enabled, but write the tests as xfail.  This means they'll be less likely to get lost.